optimum/intel/openvino/trainer.py

```diff
@@ -100,6 +119,7 @@ def __init__(
         preprocess_logits_for_metrics: Callable[[torch.Tensor, torch.Tensor], torch.Tensor] = None,
         ov_config: Optional[OVConfig] = None,
         feature: Optional[str] = None,
+        onnx_config: Optional[OnnxConfig] = None,
```

Comment:
In which case the user needs to use its own `onnx_config` ? I would remove this parameter in order to not let the possibility to have different onnx_config that we do not support during export

Reply:
@vuiseng9, can you please clarify this?

Reply:
The original implementation of [onnx_export](https://github.com/huggingface/optimum-intel/blob/main/optimum/intel/openvino/trainer.py#L562) requires an `OnnxConfig` object as input and corresponding `OnnxConfig` for a given task/architecture can be extracted from transformers' `FeatureManager` ([see](https://github.com/huggingface/optimum-intel/blob/main/optimum/intel/openvino/trainer.py#L539)). For some reasons, `Wav2vec2` has no entry in the `FeaturesManager`. In order to comply to design, we expose this to the user. We also created one for `Wav2vec2` in this pull request. 
@echarlaix Any reason why `wav2vec2` has not entry in `wav2vec2`?

Reply:
The ONNX export of wav2vec2 models was recently added to [`optimum`](https://github.com/huggingface/optimum/blob/v1.6.3/optimum/exporters/onnx/model_configs.py#L802), I opened https://github.com/huggingface/optimum-intel/pull/173 to integrate it to the `OVQuantizer` and wanted to wait the merging of https://github.com/huggingface/optimum-intel/pull/150 before adding it to the `OVTrainer` but will integrate it to https://github.com/huggingface/optimum-intel/pull/150 as it's now needed on your end

Reply:
https://github.com/huggingface/optimum-intel/pull/173 was merged, feel free to rebase !

Reply:
Rebased )